resources/build_essential.rb

```diff
@@ -62,11 +62,13 @@
       package 'bison'
       package 'gnu-coreutils'
       package 'flex'
-      package 'gcc' do
-        # lock because we don't use 5 yet
-        version '4.8.2'
+      # lock gcc versions because we don't use 5 yet
+      %w(gcc gcc-c gcc-c++).each do |pkg|
```

Comment:
I think one of the Travis failures is caused by foodcritic rule getting triggered here. I think you might have to just define each package as a separate resource.

Reply:
You can inline disable this foodcritic rule as well